react/react-addons-transition-group.d.ts

```diff
@@ -7,7 +7,7 @@
 
 declare namespace __React {
 
-    interface TransitionGroupProps {
+    interface TransitionGroupProps extends HTMLAttributes {
         component?: ReactType;
```

Comment:
I can't think of a good way. Kind of need the extended interface to be a generic, but `interface Blah<T> extends T` fails pretty spectacularly